app/controllers/api/v1/account/admin/osbbs_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::Account::Admin::OsbbsController < ApplicationController
+  include ActionController::ImplicitRender # if you need render .jbuilder
```

Comment:
maybe this should be in base API controller
for everyone to use?

Reply:
fixed